src/kartograf/filters/ring_changes.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def filter_ringsize_changes(molA, molB,
+                            mapping: dict[int, int]) -> dict[int, int]:
+    """Prevents mutating the size of rings in the mapping"""
+    riA = molA.GetRingInfo()
```

Comment:
Could there be a case, when the RingInfo was not Initialized? is there a way to do that?

Reply:
https://github.com/rdkit/rdkit/issues/1984

Reply:
reading that issue, since our molecules will be "sanitized" we should always have RingInfo() present